### PR TITLE
Add CRUD functionality into the crud.js file

### PR DESCRIPTION
**Pull request by fabianomag**
Implementation the CRUD (create, update, delete) methods

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      items.push(newEntry);
+      addEnter.value = '';
+      checkList.innerHTML = '';
+      setStored();
+      displayList();
+    }
+  });
+};
+
+const removeCrud = (trashIcon) => {
```
Comment (D4nielJ) [inline src/crud.js:20]: - [ ] When I try to remove a task from the list via the trash icon, sometimes, other tasks get removed aswell. I think this is not the intended behavior. Please, let's review the functions related and try to fix this bug.

todo_list_deletes_more_items

File: src/index.js [hunk 2]
```diff
@@ -1,29 +1,14 @@
+/* eslint-disable import/no-cycle, object-curly-newline, import/no-mutable-exports */
```
Comment (D4nielJ) [inline src/index.js:1]: I think that disabling multiple linter rules is a bad practice. The purpose of them is to keep our code clean, avoid issues and standardize code between teams. For that reason, it's important that we research anytime we're facing these to implement a better solution than just disabling them.

- [ ] Let's try to remove as much of them as possible, especially the second one and third one I think are easy to fix. The first one is a bit more tricky and requires a lot of refactor, if you can't fix it it's okay.

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @fabianofrank, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :fireworks:
- Tasks are correctly stored in localStorage
- I can add tasks, edit them and remove them in batches :clap: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Frank, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**